beep/generate_protocol.py

```diff
@@ -202,14 +205,20 @@ def generate_protocol_files_from_csv(csv_filename, output_directory=None):
             wr.writerow([name])
     outputfile.close()
 
+    num_generated_files = len(successfully_generated_files)
+    num_generation_failures = len(file_generation_failures)
+    num_files = num_generated_files + num_generation_failures
+
+    message = {
+        "comment": "Generated {} of {} protocols".format(num_generated_files, num_files),
+        "error": ""
+    }
     if not result:
         result = "success"
-        message = {
-            "comment": "Generated {} protocols".format(str(len(new_files))),
-            "error": "",
-        }
+    else:
+        message["error"] = "Failed to generate {} of {} protocols".format(num_generation_failures, num_files)
```

Comment:
Would it also make sense send this message to the logger?

Reply:
@patrickherring-TRI 
I've never consumed these logs myself so I'm not sure. Right now this appears to be logged in JSON somewhere, I thought that would be sufficient. If you'd like I could add the following line below the one here
```python
logger.error(message["error"])
```
before merging.

Reply:
Yeah, if you could add that I think it would provide some better information at the logging level. We are still trying to sort out exactly how to handle logs and other error reporting streams.